src/cli/commands/force/lightning/lwc/preview.ts

```diff
@@ -279,7 +278,39 @@ export class Preview extends Setup {
         });
     }
 
-    public async launchPreview(): Promise<void> {
```

Comment:
Didn't add or delete anything. Just make `launchPreview()` private and moved it to below all of the `protected` methods